plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -145,7 +145,16 @@
     type: str
     choices: ["never", "always", "default_off", "default_on"]
     version_added: "3.4.0"
-
+  ci_config_path:
+    description:
+      - The path to CI configuration file.
```

Comment:
Maybe add a paragraph mentioning where this path is relative to. Because it's very likely not a path on the machine where the module is executed ;-)

Reply:
How about:

- The path to CI configuration file (relative to the project).
- Defaults to `.gitlab-ci.yml`

Reply:
I would leave the default away, just in case GitLab decides to change it eventually ;-) About the first line: how about "inside" instead of "relative"? But in any case, this would be a helpful addition IMO!

Reply:
I hope you're fine with the new description, it should be clear enough ;)